README.md

```diff
@@ -41,15 +41,15 @@ IPFS Lite runs the minimal setup required to provide a DAG service. It is a port
 - [x] Stop IPFS Lite
 - [x] `addFileWithParams:input:completion:` Add data via `NSInputStream`.
 - [x] `getFileWithCid:completion:` Asynchronously get file by content address.
```

Comment:
you've diverged from the method names in the core API here, seems more sustainable to just use the same across them all

Reply:
Yea it's a tough call. If you just take the core names and as directly as possible use them in Obj c, you end up with names that are very weird and non-standard for obj c.

Reply:
Ok for now, I've made sure that at least the first part of each method name is the same as the grpc service definition methods in the Proto file. I'm also trying to be consistent with Apple's obj guidelines where you give the first method parameter a label as well. For example, `getFileWithCid`.. `getFile` is consistent with our service definition and `WithCid` provides the familiar obj c way of dealing with parameter labels.